src/components/Shared/Filters/FilterListItemMultiselect.tsx

```diff
@@ -122,7 +127,59 @@ export const FilterListItemMultiselect: React.FC<Props> = ({
             />
           </ListItem>
         ))
-      )}
+      ) : filter.filterKey === 'status' ||
```

Comment:
Do you mind adding a comment about the intention of this special case? Basically that we're grouping the status options by phase?